tools/packaging/static-build/qemu/build-static-qemu.sh

```diff
@@ -57,6 +57,7 @@ sudo docker build \
 
 sudo docker run \
 	--rm \
+	--privileged=true \
```

Comment:
why this is needed ?

Reply:
Ops, it's a leftover. I will remove it on final version of this PR.